qiskit/optimization/converters/linear_equality_to_penalty.py

```diff
@@ -112,3 +122,39 @@ def encode(self, op: QuadraticProgram, penalty_factor: float = 1e5,
             self._dst.maximize(offset, linear, quadratic)
 
         return self._dst
+
+    def _auto_define_penalty(self, penalty_factor: float = 1e5) -> float:
+        """Automatically define the penalty coefficient.
+
+        This returns (upper bound - lower bound + 1) of the objective function.
+
+        Args:
+            penalty_factor: The penalty factor from `LinearEqualityToPenalty.encode`
```

Comment:
This seems to be some input value that will be used for the returned penalty factor if the automatic computation is not done - ie the return after the logger.warning. Hence the Returns text it seems is only partially correct.